docs/04-operation-guides/troubleshooting/svls-05-serverless-periodically-restarting.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Serverless periodically restarting
+---
+
+
+## Symptom
+
+Serverless restarting every 10 minutes when reconciling git-sourced functions.
+
+## Cause
+
+Function controller is polling for changes in referenced git repositories. If you have a lot of git-sourced functions and they were deployed together in approximately the same time their git sources will be checked-out in a synchronised pulse (every 10 minutes). If you happen to reference large repositories (multi-repositories) there will be rhythmical high demand on cpu and I/O resources necessary to checkout repositories that may cause function controller to crash and restart.
+
+## Remedy
+
+Avoid using multi-repositories or large repositories to source your git functions. Using small or dedicated function repositories will decrease cpu and I/O  resources used to checkout git sources and hence assure better stability of functions controller.
```

Comment:
```suggestion
Avoid using multi-repositories or large repositories to source your git Functions. Using small, dedicated Function repositories decreases the cpu and I/O resources used to check out git sources, and hence improve the stability of Function controller.
```
Is it really "or"? I'm assuming a dedicated Function repo should still be small?